### DISCUSSION TITLE
OAuthCreateAccount error

### DISCUSSION THREAD

**Discussion by cjdunteman** [nextauthjs/next-auth#6026] [Help] [answered]
I am using Planetscale, the Prisma Adapter, and Vercel. Whenever I try to sign in I receive a "Try signing in with a different account." message. The url then looks like this: `https://cjdunteman.com/api/auth/signin?error=OAuthCreateAccount`.

![image](https://user-images.githubusercontent.com/37129024/206938635-f7e0042b-772e-46a1-a095-0b5d431756cc.png)

I've set the callback url in my GitHub app to `https://cjdunteman.com/api/auth/callback/github`.

Here is the [schema](https://github.com/cjdunteman/cjdunteman.com/blob/main/prisma/schema.prisma). I just followed the prisma adapter documentation in creating it. 

I don't have NEXTAUTH_URL set at all in Vercel re: #5624

I don't have backticks on the DATABASE_URL in Vercel re: #5127

This [link](https://github.com/nextauthjs/next-auth/blob/cbf9622e6fc18e6cb344e1b0fae3e8e39bcf5aee/docs/versioned_docs/version-v3/configuration/pages.md#sign-in-page) says that the OAuthCreateAccount error is due to a failure in creating the user in the database. I can't figure out why this is failing. 

The full repo is public if that helps: [repo](https://github.com/cjdunteman/cjdunteman.com)

**Comment by cjdunteman** [comment, marked as answer]
Got it working. Had some issues changes I made to the schema.

**Reply by mendykrauss** [reply]
I wish I would know how you fixed it

**Reply by jay-cascade** [reply]
Me too

**Comment by harshilAmbliya** [comment]
i have also same error i use postgres database and prisma and i also got it

**Comment by harshilAmbliya** [comment]
import { NextAuthOptions } from "next-auth";
import GitHubProvider from "next-auth/providers/github";
import { PrismaAdapter } from "@auth/prisma-adapter"
import prisma from "@/prisma";


const authOptions: NextAuthOptions = {
    adapter: PrismaAdapter(prisma),
    providers: [
        GitHubProvider({
            clientId: process.env.GITHUB_ID!,
            clientSecret: process.env.GITHUB_SECRET!,
            profile(profile, tokens) {
                return {
                    id: profile.id,
                    access_token: profile.access_token,
                    expires_at: profile.expires_at,
                    refresh_token: profile.refresh_token,
                    username: profile.username ?? "",
                    role:profile.role ??"user",
                }
            },
            allowDangerousEmailAccountLinking: true,
            
        })
    ],
    secret: process.env.NEXT_SECRET!,
    session: {
        strategy: "jwt"
    },
    debug: true,
    callbacks: {
        async signIn({ user, account, profile, email, credentials }) {

            console.log("Sign in", user, account, profile, email, credentials);
            return true
        },
    }
}

export default authOptions;
